sycl/test-e2e/Basic/kernel_info.cpp

```diff
@@ -4,8 +4,8 @@
 // Fail is flaky for level_zero, enable when fixed.
 // UNSUPPORTED: level_zero
 //
-// Consistently fails with opencl, enable when fixed.
-// XFAIL: opencl
+// Consistently fails with opencl non-fpga, enable when fixed.
+// XFAIL: opencl && !accelerator
```

Comment:
I'm somewhat surprised acc behaves differently from cpu... Anyway, not testable in pre-commit, so merging.